
dt/bindings: power: add RK3528 SoCs header for idle-request

According to a description from TRM, add all the idle request.

Code from BSP kernel by:
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>

Copyright (C) 2023 Piotr Oniszczuk <piotr.oniszczuk@gmail.com>

---
 include/dt-bindings/power/rk3528-power.h | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)
 create mode 100644 include/dt-bindings/power/rk3528-power.h

diff --git a/include/dt-bindings/power/rk3528-power.h b/include/dt-bindings/power/rk3528-power.h
new file mode 100644
index 000000000000..4f7c978651b5
--- /dev/null
+++ b/include/dt-bindings/power/rk3528-power.h
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: (GPL-2.0+ OR MIT) */
+#ifndef __DT_BINDINGS_POWER_RK3528_POWER_H__
+#define __DT_BINDINGS_POWER_RK3528_POWER_H__
+
+/*
+ * RK3528 idle id Summary.
+ */
+#define RK3528_PD_PMU		0
+#define RK3528_PD_BUS		1
+#define RK3528_PD_DDR		2
+#define RK3528_PD_MSCH		3
+#define RK3528_PD_GPU		4
+#define RK3528_PD_RKVDEC	5
+#define RK3528_PD_RKVENC	6
+#define RK3528_PD_VO		7
+#define RK3528_PD_VPU		8
+
+#endif
-- 
2.36.1

